Week-2/hometown webpage.html.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kolkata</title>
+</head>
+<body>
+    <h1>Kolkata, India</h1>
+    <section>
+        <h2>Location</h2>
+        <p>Kolkata also known as Calcutta (which was the official name until 2001) is the capital of the Indian state of West Bengal. 
+            <br>It is on the eastern bank of the Hooghly River 80 km (50 mi) west of the border with Bangladesh.</p>
```

Comment:
Avoid the Br tag, leverage the block level tags of HTML. Its better to use block scope elements than the br tag. Specially I am not big fan of the br tag and in the industry also, its reflecting the lack of knowledge if you use br tag very often.

Please consider this for the future, when you will be writing the production code. Thanks

Reply:
Ok I will keep this in mind. From now onwards.

Reply:
Ok I will keep this in mind. From now onwards.

Reply:
Ok I will keep this in mind. From now onwards.